esphome/components/online_image/__init__.py

```diff
@@ -152,6 +154,10 @@ async def to_code(config):
 
     cg.add(var.set_transparency(transparent))
 
+    if placeholder_id := config.get(CONF_PLACEHOLDER)
```

Comment:
Woops, I forgot this =/
```suggestion
    if placeholder_id := config.get(CONF_PLACEHOLDER):
```